client/lxd_images.go

```diff
@@ -387,7 +387,11 @@ func (r *ProtocolLXD) CreateImage(image api.ImagesPost, args *ImageCreateArgs) (
 		}
 
 		// Rootfs file
-		fw, err = w.CreateFormFile("rootfs", args.RootfsName)
+		if args.Type == "virtual-machine" {
```

Comment:
@stgraber can you use the constant here for this? https://github.com/lxc/lxd/blob/master/shared/api/instance.go#L17

Reply:
See above